lib/NodeGraphEditor.stories.tsx

```diff
@@ -11,15 +11,15 @@ const simpleConfig: IGraphConfig = {
       defaultValue: '',
     },
   },
-  nodeGroups: {
+  nodeThemes: {
     default: {
       name: 'Default',
       color: '#a1a1a1',
     },
   },
```

Comment:
This is really the only change I'm concerned about. This restricts the node group config from being used for anything but theming. While that's all it does today, I'm not sure that that will be true forever. Let me noodle on this a bit and see if I can think of any other short-term uses for the node group config that would be restricted by this change.

Reply:
Maybe NodeKind or something. Group is used for theme and group is used for input groups inside a node, and soon hopefully, group will mean a group of nodes.

Reply:
Agreed, the naming will be ambiguous with the advent of node groups. I like `nodeKinds` and the associated `kind` reference from the node configs. I think that solves the ambiguity problem without restricting what we can do with the config in the future.

Reply:
eh... you end up in the examples with things like `kind: 'geometry'` should really gets me thinking it does something else. Maybe package is better than kind. Maybe wait until something non theme comes up?